src/components/Login.js

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react'
+import { NavLink} from 'react-router-dom'
+import axios from "axios";
+
+const Login = () => {
+
+    const [user, setUser] = useState({
+        email: "",
+        password: ""
+    })
+    const handleChange = e => {
+        const { name, value } = e.target
+        setUser({
+            ...user,
+            [name]: value
+        })
+    }
+
+    const  login=()=>{
+        axios.post("http://localhost:8000/app/login",user)
+        .then(res=>alert(res))
+    }
+
+    // const history = useHistory();
+    // const [email, setEmail] = useState('');
+    // const [password, setPassword] = useState('');
+
+    // const loginUser = async (e) => {
+    //     e.preventDefault();
+
+    //     const res = await fetch('/signin', {
+    //         method: "POST",
+    //         headers: {
+    //             "Content-Type": "application/json"
+    //         },
+    //         body: JSON.stringify({
+    //             email,
+    //             password
+    //         })
+    //     });
+
+    //     const data = res.json()
+
+    //     if (res.status === 400 || !data) {
+    //         window.alert("Invalid Credentials")
+    //     }
+    //     else {
+    //         window.alert("Login Successfull")
+    //         history.push('/')
+    //     }
+
+    return (
+        <div center>
+           { console.log(user)}
+            <section className='sign-in'>
+                <div className='container mt-5'>
+                    <div className="signin-content">
+
+                        <div className='signin-image'>
+                            <figure>
+                                <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQTGWB_-boeAKTKNFnE2azbu3iblEn9L1Aa_Q&usqp=CAU" />
```

Comment:
download images locally and give theire path rather thahaving online link.
please follow for all the image paths